contracts/requirements.txt

```diff
@@ -2,5 +2,5 @@ ecdsa
 fastecdsa
 sympy
 cairo-lang==0.10.3
-starknet-devnet>=0.4.2
+starknet-devnet>=0.4.6
```

Comment:
Starknet just released https://github.com/Shard-Labs/starknet-devnet/releases 0.5.0a0 which supports starknet 0.11 and cairo 1.0 but I believe right now they only support x86